### PR TITLE
changed the reference of base docker images from dockerhub to jfrog

### PR DESCRIPTION
**Pull request by haal9000**
## Description

TODO: Describe your changes in detail here.

## Related Issue

Closes #ISSUE_NUMBER.

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### Resolved issues:
1. [x] resolves magento/pwa-studio#3751: changed the reference of base docker images from dockerhub to jfrog

### CODE REVIEW

File: venia-integration-tests/run-tests.js [hunk 1]
```diff
@@ -84,14 +84,14 @@ if (port) {
 
     dockerCommand = `docker run --rm -v ${
         process.env.PWD
-    }:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress cypress/included:8.3.1 run --browser chrome --config baseUrl=https://host.docker.internal:${port},screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=${update} --headless --reporter mochawesome --reporter-options reportDir=cypress/results,overwrite=false,html=false,json=true`;
+    }:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress docker-hub-remote.dr-uw2.adobeitc.com/cypress/included:8.3.1 run --browser chrome --config baseUrl=https://host.docker.internal:${port},screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=${update} --headless --reporter mochawesome --reporter-options reportDir=cypress/results,overwrite=false,html=false,json=true`;
```
Comment (dpatil-magento) [inline venia-integration-tests/run-tests.js:87]: Local vs CI case needs to be handled.

File: venia-integration-tests/run-tests.js [hunk 2]
```diff
@@ -84,14 +84,14 @@ if (port) {
...
 } else {
     // run docker on remote instance
     console.log(`Running tests on remote instance ${baseUrl}`);
 
     dockerCommand = `docker run --rm -v ${
         process.env.PWD
-    }:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress cypress/included:8.3.1 run --browser chrome --config baseUrl=${baseUrl},screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=${update} --headless --reporter mochawesome --reporter-options reportDir=cypress/results,overwrite=false,html=false,json=true`;
+    }:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress docker-hub-remote.dr-uw2.adobeitc.com/cypress/included:8.3.1 run --browser chrome --config baseUrl=${baseUrl},screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=${update} --headless --reporter mochawesome --reporter-options reportDir=cypress/results,overwrite=false,html=false,json=true`;
```
Comment (dpatil-magento) [inline venia-integration-tests/run-tests.js:94]: Local vs CI case needs to be handled.

**Comment by sharkySharks** [comment]
@haal9000 I don't know that we want to be changing this to use the adobe internal artifactory instance because this docker file can be used externally to build pwa as well and this will break for public users since they do not have access. I think we should keep this generic to docker hub public images, but update the credentials for the user in our CI environment. Not sure we even need to use credentials for public images...

**Comment by sharkySharks** [comment]
run lighthouse-desktop

**Comment by haal9000** [comment]
> @haal9000 I don't know that we want to be changing this to use the adobe internal artifactory instance because this docker file can be used externally to build pwa as well and this will break for public users since they do not have access. I think we should keep this generic to docker hub public images, but update the credentials for the user in our CI environment. Not sure we even need to use credentials for public images...

I believe docker has introduced some throttling on the amount of data we can pull from public docker hub without having an account(ref: https://magento.slack.com/archives/C1XJ71VN3/p1608143019296000). If we have to use public docker repo then we might need a generic paid account whose credentials we can use in our CI.

**Comment by anthoula** [comment]
@magento create issue from PR

**Pull request closed without merging**